doc/tutorials/using_cat_on_up2.rst

```diff
@@ -0,0 +1,163 @@
+.. _using_cat_up2:
+
+Using CAT on UP2
+################
+
+The UP Squared board `(UP2) <https://up-board.org/upsquared/specifications/>`_
+is an x86 maker board based on the Intel Apollo Lake platform and supports
+Cache Allocation Technology (CAT). With this feature, the usage of cache can be
+restricted to each VM. ACRN hypervisor can do that by giving each VM a cache mask,
+so that the VM will not evict the masked cache lines, as shown in :numref:
+`cache-mask-vm`:
+
+.. figure:: images/using_cat_up2.png
+   :align: center
+   :name: cache-mask-vm
+
+   Cache usage with Cache Mask
+
+
+CAT Support on ACRN
+*******************
+
+As described at ``Intel® 64 and IA-32 Architectures Software Developer’s Manual``,
+chapter 17.17, volume 3. There are 3 steps to use CAT:
```

Comment:
Isn't that 17.19 rather?

Reply:
Yes, that is 17.19, in https://software.intel.com/en-us/download/intel-64-and-ia-32-architectures-sdm-combined-volumes-3a-3b-3c-and-3d-system-programming-guide